### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by nandorojo** [Anifacted/lerna-update-wizard#124]
This package is a life saver!

**Comment by Anifacted** [comment]
Thank you for the kind words. I am happy to have contributed with something of value! ❤️

**Comment by nandorojo** [comment]
It’s amazing! I might recommend changing the name, since you don’t need lerna to use it (only a monorepo). Whenever I recommend it, people say, ah I don’t use lerna.

**Comment by Anifacted** [comment]
Yes, I have thought about that. 👍  Just not sure what the right approach would be.
I suppose I can update the repo and package name and just keep the existing commands working for backwards compatibility.
